src/client/index.ts

```diff
@@ -13,10 +13,25 @@ import {tryServiceDispatch, eventEmitter, getEventRouter} from './connection';
 import {ButtonOptions, ControlOptions} from './controls';
 import {APITopic, Events, NotificationInternal, Omit} from './internal';
 import {EventRouter, Transport} from './EventRouter';
+import * as provider from './provider';
 
 export * from './actions';
 export * from './controls';
 
+export {provider};
+
+/**
+ * The version of the NPM package.
+ *
+ * Webpack replaces any instances of this constant with a hard-coded string at build time.
+ */
+declare const PACKAGE_VERSION: string;
+
+/**
+ * The Clients' version number.
```

Comment:
Can we phrase in a way that makes more sense for someone reading the API docs?

Reply:
✅